apis/validator/blinded_block.yaml

```diff
@@ -33,14 +33,7 @@ get:
       schema:
         $ref: '../../beacon-node-oapi.yaml#/components/schemas/Graffiti'
     - name: skip_randao_verification
-      in: query
-      required: false
-      description: |
-        Skip verification of the `randao_reveal` value. If this flag is set then the
-        `randao_reveal` must be set to the point at infinity (`0xc0..00`). This query parameter
```

Comment:
Are we fine with removing the explicit note?

> This query parameter is a flag and does not take a value.

The schema now enforces flag-like behavior as it won't allow you to set a value anymore but since openapi does not natively support a flag schema (https://github.com/OAI/OpenAPI-Specification/issues/1782), being explicit aboout might be good.

Reply:
i was going to merge, but are we happy here? it sounds unresolved...

Reply:
I guess the min / max length makes it clear as well